app/views/messages/index.html.erb

```diff
@@ -21,7 +21,7 @@
     <div class="grid__item width-two-thirds shift-one-sixth">
       <%= form_for @message, remote: true, :html => {class: 'sendbar form-width--searchbar' }, builder: GcfFormBuilder do |f| %>
           <%= f.text_field(:body, { autofocus: true, type: 'text', class: 'sendbar__input', autocomplete: 'off', autocorrect: 'off', autocapitalize: 'off', placeholder: 'Send a text message' }) %>
-          <button id="send_message" class="button button--primary" type="submit">Send</button>
+          <button id="send_message" class="sendbar__button button--primary" type="submit">Send</button>
```

Comment:
<img width="322" alt="screen shot 2017-07-05 at 11 26 50 am" src="https://user-images.githubusercontent.com/1418949/27878967-eaa4809a-6174-11e7-9c83-1bf39a996a9b.png">
<img width="1440" alt="screen shot 2017-07-05 at 11 27 00 am" src="https://user-images.githubusercontent.com/1418949/27878966-eaa1f6cc-6174-11e7-8797-8d8b5aff0b7e.png">

Ok, think this works now.